### PR TITLE
[AiLab] correctly access data in testValues

### PR DESCRIPTION
**Pull request by Erin007**
tldr: "Whipped Cream" != "WhippedCream"

We got reports from the curriculum team of a bug where the prediction in AI Lab didn't match the prediction in App Lab for the same trained model with the same test values. This initially seemed very worrisome, but was actually quite a straightforward fix. 

In #39067 we striped column names from AI Lab trained models of whitespace and special characters so they could be used as design element ids and keys in the `testValues` object. 

However, in the `predict` function we were not correctly accessing values in `testValues` to pass to the `convertTestValue` function from #39044. Therefore, certain features (particularly those with spaces, e.g. "whipped cream" and "fried chicken") were being converted to NaN, which was causing the trained model to return inaccurate predictions. 

BEFORE: 
Screen Shot 2021-02-16 at 10 23 31 PM

AFTER: 
Screen Shot 2021-02-16 at 10 06 41 PM

Now, predictions are consistent across AI Lab and App Lab.

### CODE REVIEW

File: apps/src/MLTrainers.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ export function predict(modelData) {
       convertTestValue(
         modelData.featureNumberKey,
         feature,
-        modelData.testData[feature]
+        modelData.testData[feature.replace(/\W/g, '')]
```
Comment (breville) [inline apps/src/MLTrainers.js]: Since we have this snippet a few times now, can we hoist it to a helper function?

Comment (Erin007) [reply apps/src/MLTrainers.js]: Where should it live? It's used in both ai.js and here in MLTrainers.js. Just pick one and export to the other?

Comment (breville) [reply apps/src/MLTrainers.js]: I'm not too sure about the `/apps` heirarchy, but I think it makes sense for `applab/ai.js` to reach into its parent directory for such a utility function (since it seems we have a variety of "utils" and "common" files there).  It might be cleanest to add this to `apps/src/utils.js` so that we don't create a dependency between the two existing source files, or to create a new `apps/src/aiUtils.js` file?

Comment (Erin007) [reply apps/src/MLTrainers.js]: Sounds good. I went with the aiUtils option.

**Review by breville — APPROVED** [review]
